rules/windows/privilege_escalation_named_pipe_impersonation.toml

```diff
@@ -17,17 +17,66 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Privilege Escalation via Named Pipe Impersonation"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Privilege Escalation via Named Pipe Impersonation
+
+A named pipe is a type of inter-process communication (IPC) mechanism used in operating systems like Windows, which allows two or more processes to communicate with each other by sending and receiving data through a well-known point.
+
+Attackers can abuse named pipes to elevate their privileges by impersonating the security context in which they execute code. Metasploit, for example, creates a service and a random pipe, which uses the service to connect to the pipe, which impersonates the service security context, which is SYSTEM.
```

Comment:
Reducing repetition of "which" phrases -- please confirm whether this changes any of the description's meaning/accuracy:
```suggestion
Attackers can abuse named pipes to elevate their privileges by impersonating the security context in which they execute code. Metasploit, for example, creates a service and a random pipe, and then uses the service to connect to the pipe and impersonate the SYSTEM service security context.
```